packages/plugin-packer/src/create-contents-zip.ts

```diff
@@ -0,0 +1,32 @@
+import path from "path";
+import { ZipFile } from "yazl";
+import streamBuffers from "stream-buffers";
+import { sourceList } from "./sourcelist";
+import _debug from "debug";
+
+const debug = _debug("create-contents-zip");
+
+/**
+ * Create a zipped contents
+ */
+export function createContentsZip(
+  pluginDir: string,
+  manifest: any
+): Promise<false | Buffer> {
```

Comment:
I understand that this patch can't apply because `streamBuffers.WritableStreamBuffer()` returns false or Buffer.
However, I think it is a bug that WritableStreamBuffer() returns false.
The [issue](https://github.com/samcday/node-stream-buffer/issues/33) and [PR](https://github.com/samcday/node-stream-buffer/pull/42) are created. But `stream-buffers` seems not to be maintained.
Do you think that this function should false or not?

```suggestion
): Promise<Buffer> {
```

Reply:
I see. Ideally `stream-buffer` should be fixed, but I think this is enough for now 7fadf99